runtime/hsa-runtime/inc/hsa_ext_amd.h

```diff
@@ -2813,6 +2813,93 @@ typedef enum hsa_amd_queue_priority_s {
 hsa_status_t HSA_API hsa_amd_queue_set_priority(hsa_queue_t* queue,
                                                 hsa_amd_queue_priority_t priority);
 
+/**
```

Comment:
Need to bump HSA_AMD_INTERFACE_VERSION_MINOR

Reply:
Done